openwrt-patch: add kernel modules for rtl8150 and rtl8152 based usb-t…

A-Kasper opened the pull request:
Kernel: Realtek 8150 and Realtek 8152 based USB-to-Ethernet converters

This patch adds the kernel modules to for the very common Realtek RTL8150 and Realtek 8152 based USB-to-Ethernet converters. 
After this patch the following packages can be added to targets (or site.mk):
kmod-usb-net-rtl8150
kmod-usb-net-rtl8152

The modules are already available in openwrt trunk, but not in Chaos Calmer. This patch is just a temporary backport.
The modules are needed for Futros with additional USB-Ethernet-Interface and VMs with dedicated USB-NIC

NeoRaider commented:
Mmh... did you create the patch using `make update-patches`? The filename is weird (has two dashes after the patch number), and there are some very strange mistakes in the patch file content ("Date: Thu, 12 Oct 2015 02:31:05 +0200" instead of "Date: Mon, 12 Oct 2015 02:31:05 +0200").

I've fixed up the mistakes and merged the fixed version.

A-Kasper commented:
I made it with git diff. I added the Header by hand and tried to stay in
same format as your patches. The strange date thing was my fault. for the
next time:
how do you want patches to be created?
Am 12.10.2015 11:57 schrieb "Matthias Schiffer" <notifications@github.com>:

> Mmh... did you create the patch using make update-patches? The filename
> is weird (has two dashes after the patch number), and there are some very
> strange mistakes in the patch file content ("Date: Thu, 12 Oct 2015
> 02:31:05 +0200" instead of "Date: Mon, 12 Oct 2015 02:31:05 +0200").
>
> I've fixed up the mistakes and merged the fixed version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freifunk-gluon/gluon/pull/521#issuecomment-147355439>.
>

tcatm commented:
Add the commit to the `patched` branch (of OpenWrt), the run `make update-patches`, add the resulting, new patch, and commit again (this time to gluon).
